src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,12 @@
 const booleanToWord = (boolean) => {
-
-}
-
+    if (boolean === true) { /*explicit check to make sure no none relevant
+                              values submitted*/
+        return 'yes';
+    } else if (boolean === false) {
+        return 'no';
+    } else {
+      return 'error';
+    }
```

Comment:
I would probably avoid an `if / else if / else` statement here as each outcome returns you out of the function. Instead you could just do a couple of `if` statements, and return at the end if neither were true:

```js
if (boolean === true) {
  return 'yes';
}

if (boolean === false) {
  return 'no';
}

return 'error';
```